merlin/analysis/decode.py

```diff
@@ -20,6 +21,14 @@ def __init__(self, dataSet: dataset.DataSet, parameters=None,
                  analysisName=None):
         super().__init__(dataSet, parameters, analysisName)
 
+        if 'remove_z_duplicated_barcodes' not in self.parameters:
```

Comment:
Can this move to the Decode task? Many other tasks, such as the filter tasks are BarcodeSavingParallelAnalysisTasks but don't need these parameters.